nearcore/res/example-config-gc.json

```diff
@@ -24,7 +24,8 @@
     },
     "telemetry": {
         "endpoints": [
-            "https://explorer.mainnet.near.org/api/nodes"
+            "https://explorer.mainnet.near.org/api/nodes",
+            "https://telemetry.nearone.org/nodes/mainnet"
```

Comment:
how are these multiple endpoints used by the code? the first one is primary and the other is fallback or does it send requests to both?

Reply:
the requests are sent to both (see the [actor implementation](https://github.com/near/nearcore/blob/master/chain/telemetry/src/lib.rs#L80))